app/views/shared/_plausible_script.html.erb

```diff
@@ -0,0 +1,20 @@
+<% if ENV["PLAUSIBLE_DOMAIN"] && ENV["PLAUSIBLE_DOMAIN"].include?("staging") %>
```

Comment:
Not sure why we need the second part of the check to see that the domain contains the word 'staging'? I think if the environment variable is set we can probably assume that we want the script to run.

Reply:
I wanted to account for that but forgot the variable in test but forgot it is only present in the staging variables. I will remove.